### PR TITLE
fastfec 0.1.9

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Requires a patch for bottling. Works for me locally; ~~will upstream if it works in CI~~.

### CODE REVIEW

**Comment by carlocab** [comment]
Linux is failing because of https://github.com/ziglang/zig/issues/14111.

File: Formula/fastfec.rb [hunk 1]
```diff
@@ -16,29 +17,40 @@ class Fastfec < Formula
...
-  depends_on "pcre"
-  uses_from_macos "curl"
+
+  on_macos do
+    # Zig attempts to link with `libpcre.a` on Linux.
+    # This fails because it was not compiled with `-fPIC`.
+    # Use Homebrew PCRE on Linux when upstream resolves
+    #   https://github.com/ziglang/zig/issues/14111
+    # Don't forget to update the `install` method.
+    depends_on "pcre"
```
Comment (SMillerDev) [inline Formula/fastfec.rb:27]: Can it use pcre2?

Comment (carlocab) [reply Formula/fastfec.rb:27]: Unfortunately not.

Comment (SMillerDev) [reply Formula/fastfec.rb:27]: If there isn't an issue yet, could you make one for that too?

**Review by SMillerDev — APPROVED** [review]

File: Formula/fastfec.rb (hunk 1 above)
Comment (carlocab) [reply Formula/fastfec.rb:27]: https://github.com/washingtonpost/FastFEC/issues/57

**Pull request closed without merging**